unstable/wlr-output-management-unstable-v1.xml

```diff
@@ -257,6 +257,55 @@
         resources associated with it.
       </description>
     </event>
+
+    <event name="make" since="2">
```

Comment:
Sometimes this information is unavailable (missing from the EDID, or virtual output). Maybe we should make the event optional?

Reply:
I would have said in this case it's just not sent. Do you mean that? So I should mention it in the description?

Reply:
Yes!

Reply:
I added some text about it to the description.  Check it out if you see it fitting. There is a "strong" statement about guaranteed uniqueness of head identification which I think makes sense when one has make, model and serial number but one should take one moment to think about corner cases.

Reply:
It's not uncommon to have broken EDIDs with the same (make, model, serial) tuple for different monitors.

Reply:
How about:

> It is not guaranteed this event will be ever sent. A reason for that can be that the compositor does not have information about the make of the head or the definition of a make is not sensible in the current setup, for example in a virtual session. Clients can still try to identify the head by available information from other events but should be aware that there is an increased risk of false positives.

Reply:
LGTM!